stdlib/end_to_end_test.go

```diff
@@ -0,0 +1,81 @@
+package stdlib_test
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"testing"
+	"time"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/querytest"
+)
+
+func runEndToEnd(t *testing.T, querier *querytest.Querier, pkgs []*ast.Package) {
```

Comment:
Do these functions need to be exported?

Reply:
At the moment no. Only if they are needed in influxDB.

Reply:
What is the plan for using the tests in InfluxDB? Is it to write their own runEndToEnd equivalent and just grab the list of test cases?

Reply:
Yes I think so. We can generate a file with the list of imports in flux. Then influxDB just grabs that list, imports the packages, and then runs all of the tests for each import with its own equivalent of `runEndToEnd`.